integration/file_Encrypted.test.js

```diff
@@ -0,0 +1,29 @@
+const { SkynetClient, defaultPortalUrl } = require("../index");
```

Comment:
I would name this `encryptedfile.test.js`, same as in `skynet-js`.

Reply:
hello m-cat,

No problem, I'll rename it.